lib/r10k/cli/version.rb

```diff
@@ -12,7 +12,16 @@ def self.command
         summary 'Print the version of r10k'
 
         run do |opts, args, cmd|
-          puts R10K::VERSION
+          puts "r10k #{R10K::VERSION}"
+          if opts[:verbose]
+            puts RUBY_DESCRIPTION
+            cmdpath = caller.last.match(/\A(.*#{$PROGRAM_NAME}):/).captures.first
+            puts "Command path: #{cmdpath}"
+            puts "Interpreter path: #{Gem.ruby}"
+            if RUBY_VERSION >= '1.9.3'
+              puts "Default encoding: #{Encoding.default_external.name}"
```

Comment:
That's the output of `Encoding#inspect`; `Encoding.to_s` prints a much prettier version:

```
[4] pry(main)> "#{Encoding.default_external}"
=> "UTF-8"
[5] pry(main)> $ Encoding.default_external.to_s

From: encoding.c (C Method):
Owner: Encoding
Visibility: public
Number of lines: 5

static VALUE
enc_name(VALUE self)
{
    return rb_usascii_str_new2(rb_enc_name((rb_encoding*)DATA_PTR(self)));
}
```

I'm debating whether the more terse form is more or less clear.